templates/frame.haml

```diff
@@ -203,6 +203,9 @@
 
       function displayAlert(type, text) {
         $('.content').prepend('<div class="alert alert-' + type + '"><a class="close" data-dismiss="alert">×</a>' + text + '</div>');
+        $(".alert").delay(4000).fadeOut(500, function() {
+          $(this).alert('close');
+        });
```

Comment:
This hides all alerts after 4 seconds, right? Is this something that users have requested?